web-marketplace/src/pages/BuyCredits/hooks/useFetchUserBalance.tsx

```diff
@@ -0,0 +1,42 @@
+import { useMemo } from 'react';
+import { useQuery } from '@tanstack/react-query';
+
+import { useLedger } from 'ledger';
+import { microToDenom } from 'lib/denom.utils';
+import { getAllBalancesQuery } from 'lib/queries/react-query/cosmos/bank/getAllBalancesQuery/getAllBalancesQuery';
+import { useWallet } from 'lib/wallet/wallet';
+
+export const useFetchUserBalance = (
+  askDenom?: string,
+): {
+  isLoading: boolean;
+  userBalance: number;
+  microUserBalance: number;
+} => {
```

Comment:
FYI it's fine to not define fn return values when we don't need to explicitly define them as it's the case here